Initium-Odp/war/odp/javascript/script.js

```diff
@@ -563,6 +563,23 @@ function buyHouse()
 	});
 }
 
+function renamePlayerHouse(eventObject)
+{
+	promptPopup("Rename Player House", "Enter a new name for your house:", "", function(newName) {
+		if (newName != null && newName != "")
+		{
+			doCommand(eventObject, "RenamePlayerHouse", {"newName" : newName});
+		}
+	});
+}
+
+function deletePlayerHouse(eventObject)
```

Comment:
So with the new concept I mentioned above, we would include a pathId to "forget".